server/app/app.py

```diff
@@ -353,32 +363,40 @@ def __init__(self, app_config):
         self.app.register_blueprint(webbp)
 
         api_version = "/api/v0.2"
+        backend_base_url = server_config.get_backend_base_url()
+        backend_path = "/"
+        if backend_base_url:
+            parse = urlparse(backend_base_url)
+            backend_path = parse.path
+
         if app_config.is_multi_dataset():
             # NOTE:  These routes only allow the dataset to be in the directory
             # of the dataroot, and not a subdirectory.  We may want to change
             # the route format at some point
             for dataroot_dict in server_config.multi_dataset__dataroot.values():
                 url_dataroot = dataroot_dict["base_url"]
                 bp_api = Blueprint(
-                    f"api_dataset_{url_dataroot}", __name__, url_prefix=f"/{url_dataroot}/<dataset>" + api_version
+                    f"api_dataset_{url_dataroot}", __name__,
+                    url_prefix=f"{backend_path}/{url_dataroot}/<dataset>" + api_version
                 )
                 resources = get_api_resources(bp_api, url_dataroot)
                 self.app.register_blueprint(resources.blueprint)
                 self.app.add_url_rule(
-                    f"/{url_dataroot}/<dataset>/",
+                    f"{backend_path}/{url_dataroot}/<dataset>/",
```

Comment:
Actually ditto here: you want the dataset to be accessible right at the frontend path, not the API path right?

Reply:
Yes, I got carried away :). fixed.